packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -305,7 +305,7 @@ class ResizeHandle extends Widget<ResizeHandleOptions> {
     this.setAria({
       role: 'application',
       // eslint-disable-next-line spellcheck/spell-checker
-      roledescription: 'separator',
+      roledescription: messageLocalization.format('dxSplitter-ariaRoleDescription'),
```

Comment:
While we're localizing `separator` not a "splitter" itself I think we should show it explicitly in the variable's name. Maybe smth like this?
```suggestion
      roledescription: messageLocalization.format('dxSplitter-separatorAriaRoleDescription'),
```

Reply:
renamed